scripts/encryption_validator/readme.md

```diff
@@ -0,0 +1,25 @@
+
+# Salt Compare Tool
+
+A lightweight Python script to compare encrypted and unencrypted salt files stored in an S3 bucket.
+
+## Description
+
+This script fetches two S3 objects (e.g. encrypted and unencrypted salts) and compares them. It validates basic input rules like prefix formatting and key pattern before proceeding.
+
+## Usage
+
+### Run the script:
+
+Login to AWS account
+`pip install requirements.txt` 
+`python script.py <key> <bucket> <region_name> [prefix]` 
+
+-   `key` – Required. Must start with `salt`. Example: `salts/salts.txt.1744152738842`
```

Comment:
should key be renamed to object? key could be confused with UID2 (API/cloud encryption/etc) keys

Reply:
@abuabraham-ttd still think you should rename key to something more specific